src/librbd/mirror/snapshot/Types.h

```diff
@@ -36,28 +36,24 @@ struct ImageStateHeader {
 WRITE_CLASS_ENCODER(ImageStateHeader);
 
 struct SnapState {
-  uint64_t id = CEPH_NOSNAP;
```

Comment:
@dillaman The reason why I initially added the `id` field was that I was thinking to use it for remote-local snap_seq mapping. I.e. on the primary site it would be the same as the index, but on the secondary site, the rbd-mirror would set this field to the local snap id.

But as you decided to store the snap_seq mapping in the snapshot namespace, it is not relevant any more.

Reply:
Gotcha -- yeah, we needed to store snap mappings for all snapshots between mirror snapshots so I needed `snap_seqs`.